### PR TITLE
kill case label and referral label

### PR DESCRIPTION
**Pull request by orangejenny**
https://manage.dimagi.com/default.asp?254069

Hold off on merge until I've finished confirming with mobile team that this is okay to kill.

code buddy @esoergel 
app manager @snopoke / @dannyroberts

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js [hunk 1]
```diff
@@ -578,10 +578,10 @@ hqDefine('cloudcare/js/backbone/apps.js', function () {
                     }
                 } else {
                     // Construct a button which will take the user to the next list of cases
-                    // Note: The case_label should be plural for this button e.g. "View Mothers"
+                    // Note: The noun should be plural for this button e.g. "View Cases"
                     //       But, in the currently selected case heading we want a singular version
                     //       e.g. "Mother: Mary"
-                    var buttonText = "View " + self.selectedModule.get("case_label")[self.options.language];
+                    var buttonText = "View Cases";
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js]: 'buttonText' is already defined (no-redeclare)

**Comment by orangejenny** [comment]
tests failures are real, will fix

**Comment by dimagimon** [comment]
There are 6 errors:

* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 117 - E501 line too long (158 > 115 characters)
* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 143 - E121 continuation line under-indented for hanging indent
E501 line too long (119 > 115 characters)
* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 171 - E501 line too long (180 > 115 characters)
* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 178 - E501 line too long (119 > 115 characters)
* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 318 - E501 line too long (122 > 115 characters)
* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 325 - E501 line too long (135 > 115 characters)

**Review by dannyroberts — APPROVED** [review]
Oh, man, yes.

I added this in the first pass of the app builder in 2010 because it happened to reflect something in the suite file I was working backwards from. Definitely double-check, but my understanding is that this hasn't been useful or meaningful for a long, long time.

**Comment by dimagimon** [comment]
There are 6 errors:

* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 117 - E501 line too long (158 > 115 characters)
* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 143 - E121 continuation line under-indented for hanging indent
E501 line too long (119 > 115 characters)
* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 171 - E501 line too long (180 > 115 characters)
* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 178 - E501 line too long (119 > 115 characters)
* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 318 - E501 line too long (122 > 115 characters)
* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 325 - E501 line too long (135 > 115 characters)

**Comment by dimagimon** [comment]
There are 6 errors:

* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 117 - E501 line too long (158 > 115 characters)
* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 143 - E121 continuation line under-indented for hanging indent
E501 line too long (119 > 115 characters)
* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 171 - E501 line too long (180 > 115 characters)
* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 178 - E501 line too long (119 > 115 characters)
* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 318 - E501 line too long (122 > 115 characters)
* corehq/apps/app_manager/tests/test_bulk_app_translation.py, line 325 - E501 line too long (135 > 115 characters)

**Comment by orangejenny** [comment]
Updated so that non-case, non-referral details (product lists and careplans) use a blank string for the detail title (previously they'd been creating an empty Text node, which parses correctly but I'm not certain will work on those rare occasions when the title is accessed).

**Comment by snopoke** [comment]
Awesome. Nice deletion / addition ratio. Amazing how complex such a 'small' feature can be.

**Pull request merged**